selfdrive/car/hyundai/values.py

```diff
@@ -2057,6 +2073,7 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
   # which message has the gear. hybrid and EV use ELECT_GEAR
   "use_cluster_gears": {CAR.ELANTRA, CAR.KONA},
   "use_tcu_gears": {CAR.KIA_OPTIMA_G4, CAR.KIA_OPTIMA_G4_FL, CAR.SONATA_LF, CAR.VELOSTER, CAR.TUCSON},
+  "use_ems_gears": {CAR.NEXO_1ST_GEN, },
```

Comment:
Just use `FCEV_CAR`, trying to move away from many hard-coded lists

Reply:
Done.